app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_parts = command.split(" ")
+
+    if not command_parts[0] or command_parts[0] != "cp":
```

Comment:
You should check that `command_parts` contains 3 elements

Reply:
Done